src/components/VideoCard/VideoCard.css

```diff
@@ -0,0 +1,38 @@
+.video-card {
+  height: 300px;
+  width: 360px;
+}
+.card-title {
+  font-weight: bold;
+  font-size: 1rem;
+  color: var(--text-light);
+}
+.card-text {
+  display: flex;
+  flex-direction: column;
+  gap: 3px;
+}
+.video-card-footer {
+  display: flex;
+  width: 100%;
+  gap: 5px;
+}
+.text-muted {
+  font-size: 0.8rem;
+}
+.video-card-channel {
+  width: 36px;
+  height: 36px;
+  border-radius: 50%;
+}
+.video-views::after {
+  content: '•';
+  margin: 0 5px;
+}
+.video-menu {
+  visibility: hidden;
+  margin-left: auto;
+}
+.video-card:hover .video-menu {
+  visibility: visible;
+}
```

Comment:
There is no hover state for mobile devices or tablets. You can remove this and make it visible for devices below 768px